applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -53,32 +64,26 @@
 #else
 #error Unsupported LC3 codec preset for unicast
 #endif /* CONFIG_BT_AUDIO_UNICAST_RECOMMENDED */
-#endif /* CONFIG_AUDIO_SOURCE_USB */
 #endif /* CONFIG_TRANSPORT_CIS */
 
 #if CONFIG_TRANSPORT_BIS
 #define BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(_loc, _stream_context)                           \
 	BT_AUDIO_LC3_PRESET(                                                                       \
-		BT_CODEC_LC3_CONFIG(                                                               \
-			BT_CODEC_CONFIG_LC3_FREQ_48KHZ, BT_CODEC_CONFIG_LC3_DURATION_10, _loc,     \
-			LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 1, _stream_context),         \
-		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 4u,     \
-					     20u, LE_AUDIO_PRES_DELAY_US))
-
-#if CONFIG_AUDIO_SOURCE_USB
-#if !CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED
-#error USB only supports 48kHz
-#endif /* !CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED */
-#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
-	BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                    \
-						  BT_AUDIO_CONTEXT_TYPE_MEDIA)
+		BT_CODEC_LC3_CONFIG(BT_AUDIO_CODEC_CONFIG_FREQ, BT_CODEC_CONFIG_LC3_DURATION_10,   \
+				    _loc, LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE), 1,    \
+				    _stream_context),                                              \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE),    \
+					     4u, 20u, CONFIG_BT_AUDIO_PRESENTATION_DELAY_US))
 
-#elif CONFIG_AUDIO_SOURCE_I2S
 #if CONFIG_BT_AUDIO_BROADCAST_RECOMMENDED
 #define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
 	BT_AUDIO_LC3_BROADCAST_PRESET_RECOMMENDED(BT_AUDIO_LOCATION_FRONT_LEFT,                    \
 						  BT_AUDIO_CONTEXT_TYPE_MEDIA)
 
+#elif CONFIG_BT_AUDIO_BROADCAST_CONFIGURABLE
+#define BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                \
+	BT_AUDIO_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
```

Comment:
We should discuss how we use BT_AUDIO_LC3_BROADCAST_PRESET_NRF5340_AUDIO and BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO.

E.g. we are setting BT_AUDIO_LOCATION_FRONT_LEFT here, even if we stream left and right audio from the same broadcast source.

Reply:
Yes, that actually came up at UPF as well. We should probably change to left and right in the base and then add left or right to the metadata of the BIS/CISes respectfully. That's a separate task though

Reply:
+1

Reply:
Who creates the task?

Reply:
Would that be OCT-2400?

Reply:
> Who creates the task?

Would that be OCT-2400?

Reply:
Yes, that seems to be OCT-2400. 
BT_AUDIO_LOCATION_FRONT_LEFT is inputted to the codec part of the preset, which is given to bt_audio_stream_config() in  &lc3_preset_nrf5340.codec.
OCT-2400 only mentions CIS and not BIS though.

Reply:
OCT-2400 should be updated